static/app/components/interactionStateLayer.tsx

```diff
@@ -7,6 +7,7 @@ import {defined} from 'sentry/utils';
 interface StateLayerProps extends React.HTMLAttributes<HTMLSpanElement> {
   as?: React.ElementType;
   color?: string;
+  hasSelectedBackground?: boolean;
```

Comment:
Since it isn't obvious, could you add a `/** */` comment here explaining what this does?